lib/place.ml

```diff
@@ -172,51 +176,53 @@ let print_html_places_surnames conf base (array : (string list * (string * iper
   loop [] (Array.to_list array) ;
   Wserver.printf "</ul>\n"
 
-let print_all_places_surnames_short conf base ~add_birth ~add_baptism ~add_death ~add_burial =
+let print_aux_opt ~add_birth ~add_baptism ~add_death ~add_burial ~add_marriage =
+    (if add_birth then "&bi=on" else "") ^
+    (if add_baptism then "&bp=on" else "") ^
+    (if add_death then "&de=on" else "") ^
+    (if add_burial then "&bu=on" else "") ^
+    (if add_marriage then "&ma=on" else "")
+
+let print_aux conf title fn =
+  Hutil.header conf title;
+  Hutil.print_link_to_welcome conf true;
+  fn () ;
+  Hutil.trailer conf
+
+let print_all_places_surnames_short conf base ~add_birth ~add_baptism ~add_death ~add_burial ~add_marriage =
   let array =
     get_all
-      conf base ~add_birth ~add_baptism ~add_death ~add_burial
+      conf base ~add_birth ~add_baptism ~add_death ~add_burial ~add_marriage
       "" 0
       fold_place_short
       (fun _ -> true)
       (fun prev _ -> match prev with Some n -> n + 1 | None -> 1)
       (fun x -> x)
+      max_int
   in
-  let title _ = Wserver.printf "%s" (capitale (transl conf "place")) in
   Array.sort (fun (s1, _) (s2, _) -> Gutil.alphabetic_order s2 s1) array ;
```

Comment:
Could you correct alphabetic order for short display ?
`Array.sort (fun (s1, _) (s2, _) -> Gutil.alphabetic_order s1 s2) array ;`

Reply:
Et bien oui, ça a l'air à l'envers...